### ISSUE TITLE
anyone know of an encoder for js? 

### ISSUE THREAD

**Issue by bj97301** [atomicobject/heatshrink#83]
I have a backend that uses this to decode and no options on my js frontend for encoding.  Just wondering if there's something for js that will encode.

**Comment by Zmmfly** [comment]
@bj97301  Hi, you can use wasm to compile this library and let it run on the browser.

**Comment by bj97301** [comment]
Good idea but I am using react native. Compiling with the browser is not so
easy.
